src/handlers/RF24StreamLogHandler.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file RF24StreamLogHandler.h
+ *
+ * Created on: 2 Oct 2020
+ *     Author: Witold Markowski (wmarkow)
+ *
+ * Copyright (C)
+ *    2020        Witold Markowski (wmarkow)
+ *
+ * This General Public License does not permit incorporating your program into
+ * proprietary programs.  If your program is a subroutine library, you may
+ * consider it more useful to permit linking proprietary applications with the
+ * library.  If this is what you want to do, use the GNU Lesser General
+ * Public License instead of this License.
+ */
+
+#ifndef SRC_HANDLERS_RF24STREAMLOGHANDLER_H_
+#define SRC_HANDLERS_RF24STREAMLOGHANDLER_H_
+
+#include <RF24LogHandler.h>
+#include <Print.h>
+
+/**
+ * A log handler implementation which outputs log messages to a stream.
+ */
+class RF24StreamLogHandler : public RF24LogHandler
+{
+protected:
+   Stream *stream;
+
+private:
+   void appendTimestamp();
+   void appendLogLevel(uint8_t logLevel);
+   void appendVendorId(const __FlashStringHelper *vendorId);

```

Comment:
Maybe those methods should be also protected, so it would be possible to override them in the subclass to provide a custom log format.

Reply:
more food for thought (from a python background):

If someone wants to change the format of the message (including timestamp, level name, and vendorId), then would it be possible to just set a format (c-)string/object that would allow this without rolling-your-own handler ([python's logging lib does this](https://docs.python.org/3/library/logging.html#formatter-objects)).